src/SFA.DAS.ApprenticeCommitments.Jobs.AcceptanceTests/Steps/NewApprenticeshipSteps.cs

```diff
@@ -3,41 +3,62 @@
 using SFA.DAS.ApprenticeCommitments.Jobs.AcceptanceTests.Services;
 using SFA.DAS.CommitmentsV2.Messages.Events;
 using System.Linq;
+using System.Net;
 using System.Threading.Tasks;
+using AutoFixture;
+using Newtonsoft.Json;
+using NUnit.Framework.Internal.Commands;
 using TechTalk.SpecFlow;
 using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
 
 namespace SFA.DAS.ApprenticeCommitments.Jobs.AcceptanceTests.Steps
 {
     [Binding]
     public class NewApprenticeshipSteps : StepsBase
     {
+        private Fixture _fixture;
+        private ApprenticeshipCreated2Event _event;
+
         public NewApprenticeshipSteps(TestContext testContext) : base(testContext)
         {
+            _fixture = new Fixture();
+            _event = _fixture.Build<ApprenticeshipCreated2Event>().With(x=>x.Email, "bob@bobs.com").Create();
+        }
+
+        [Given(@"outer api is available")]
+        public void GivenOuterApiIsAvailable()
+        {
+            testContext.Api.MockServer.Given(
```

Comment:
I'm not generally a fan of over-specifying mocks before-hand.  I prefer to assert on the interractions after the operating being tested has executed.

Let's put this through as it's written and get together as a team to discuss mocking patterns.

Reply:
yes but the tests which preceded this weren't actually testing anything and because of teh disposal  problems which existed were wouldn't have been able to tests. But lets discuss this as you say a little later